Modules/Sources/WordPressUI/Views/ScaledImage.swift

```diff
@@ -0,0 +1,21 @@
+import SwiftUI
+
+/// A simple view for scaling the given icon to the target size.
+public struct ScaledImage: View {
```

Comment:
This is more of a work-in-progress implementation. I expect it to be replaced with custom SF Symbols in the future or something else.